### PR TITLE
A New Constitution

### PR DESCRIPTION
**Pull request by jimmyhmiller**
Hear ye! Hear ye! A new constitution is upon us!!

### CODE REVIEW

**Comment by ArchmageInc** [comment]
Yay

File: laws.md [hunk 1]
```diff
@@ -1,18 +1,14 @@
 #Initial Set of Rules
 
-##Immutable Rules
+**101.** All players must always abide by all the rules then in effect, in the form in which they are then in effect. The rules in the Initial Set are in effect whenever a game begins.
 
-**101.** All players must always abide by all the rules then in effect, in the form in which they are then in effect. The rules in the Initial Set are in effect whenever a game begins. The Initial Set consists of Rules 101-116 (immutable) and 201-213 (mutable).
+**102.** No rule may make it possible for a player to win the game or end the the game.
```
Comment (dilbertKocik) [inline laws.md]: I believe the word "the" appears twice in a row here.

File: laws.md [hunk 2]
```diff
@@ -20,30 +16,28 @@
...
 
-If a rule is repealed and reenacted, it receives the number of the proposal to reenact it. If a rule is amended or transmuted, it receives the number of the proposal to amend or transmute it. If an amendment is amended or repealed, the entire rule of which it is a part receives the number of the proposal to amend or repeal the amendment.
+If a rule is repealed and reenacted, it receives the number of the proposal to reenact it. If a rule is amended it receives the number of the proposal to amend  it. If an amendment is amended or repealed, the entire rule of which it is a part receives the number of the proposal to amend or repeal the amendment.
 
-**109.** Rule-changes that transmute immutable rules into mutable rules may be adopted if and only if the vote is unanimous among the eligible voters. Transmutation shall not be implied, but must be stated explicitly in a proposal to take effect.
+**109.** A proposal must be voted on by at least 50% of active players (a quorum) in order be resolved (accepted or rejected) unless otherwise noted in the rules. Any time after a rule-change has reached a quorum it may be resolved by its creator. A rule-change is a proposal.
```
Comment (dilbertKocik) [inline laws.md]: Missing the word "to" in "in order be resolved."

File: laws.md (hunk 1 above)
Comment (jimmyhmiller) [inline laws.md]: Fixed

File: laws.md (hunk 2 above)
Comment (jimmyhmiller) [inline laws.md]: Fixed

**Comment by jimmyhmiller** [comment]
John Hancock

**Comment by ArchmageInc** [comment]
@dilbert88 Can we git this going?

**Comment by dilbertKocik** [comment]
I approve.

**Pull request merged**